core/server/data/db/backup.js

```diff
@@ -17,6 +17,21 @@ writeExportFile = function writeExportFile(exportResult) {
     return fs.writeFile(filename, JSON.stringify(exportResult.data)).return(filename);
 };
 
+const readBackup = async (filename) => {
+    const parsedFileName = path.parse(filename);
+    const sanitized = `${parsedFileName.name}${parsedFileName.ext}`;
```

Comment:
Added this sanitization to prevent path traversal attacks. @kevinansfield lmk if this seems like a reasonable approach. Wasn't able to think of a simpler way to do it :)

Reply:
Hmm, I was thinking of the endpoint using `require('express').static` with our admin auth middleware in front. I think that gives us the path traversal protection and removes the need to read/parse the file before serving it?

Reply:
I think having an endpoint for static files like this might be a bit too much of an effort to what we are trying to do here :sweat_smile: 

If we are to introduce a mechanism like that, we'd need to establish some boundaries of what exactly such endpoint would serve and think of a bigger picture like maybe we could serve other files `routes.yaml` and `redirects.json` in a similar fashion?

At this point of time the budget was 1 day for whole improvement :grimacing: so think a little patch like this is more appropriate. Wdyt?